work_queue/src/work_queue.c

```diff
@@ -1583,11 +1583,11 @@ static work_queue_result_code_t get_result(struct work_queue *q, struct work_que
 	timestamp_t effective_stoptime = 0;
 	time_t stoptime;
 
-	//Format: task completion status, exit status (exit code or signal), output length, execution time, taskid
-	char items[4][WORK_QUEUE_PROTOCOL_FIELD_MAX];
-	int n = sscanf(line, "result %s %s %s %s %" SCNd64, items[0], items[1], items[2], items[3], &taskid);
+	//Format: task completion status, exit status (exit code or signal), output length, execution time, loop device, taskid
+	char items[5][WORK_QUEUE_PROTOCOL_FIELD_MAX];
+	int n = sscanf(line, "result %s %s %s %s %s %" SCNd64, items[0], items[1], items[2], items[3], items[4], &taskid);
```

Comment:
When changing a WQ message, do it in  backwards-compatible way, since users always manage to mix and match masters and workers from different versions.   Add the new element to the end of the string, and have the master accept both n=5 and n=6.